### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by paulmorio**
## What is the goal of this PR?

Updates the Readme to reflect recent API updates

## What are the changes implemented in this PR?

The text has been reviewed and updated to reflect the new changes in the API and our focus on the 5 main modules.

The example has also been updated to reflect this

Citation information was added to the Arxiv paper

Please let me know of any discrepancies or additional information to add in.

### CODE REVIEW

**Comment by alicerdv** [comment]
Looks good! A couple of small points: Greedy Score is referenced under informativeness measures included -> should this be changed to Mean Prediction Score? Also, under - examples showcasing different uncertainty estimator, `gpytorch_integration.py` is referenced. Should this be` model_gaussianprocesses.py`?

File: README.md [hunk 1]
```diff
@@ -28,36 +33,49 @@ pip install pyrelational
...
+The AL **Strategy** (`pyrelational.strategies.abstract_strategy.Strategy`) defines an active learning strategy. We like to typically think of strategies being compositions of an *informativeness measure* and a *query selection algorithm* that selects observations based on the perceived informativeness. Together they compute the utility of a query or set of queries for a batch active mode strategy. We define various classic strategies for classification, regression, and task-agnostic scenarios based on the informativeness measures defined in `pyrelational.informativeness`. The flexible nature of the `Strategy` allows for the construction of strategies from simple serial uncertainty sampling approaches to complex agents that leverage several informativeness measures, state and learning based query selection algorithms, with query batch building bandits under uncertainty from noisy oracles. Users can implement their own strategies by overriding the `.__call__()` method. Look at `examples/demo/model_badge.py` for an example.
```
Comment (alicerdv) [inline README.md:74]: Should the import path here be pyrelational.strategies.Strategy? Same with the import path for the oracle (pyrelational.oracles.Oracle)

Comment (paulmorio) [reply README.md:74]: I agree using the shorter path looks neater.

**Comment by paulmorio** [comment]
> Looks good! A couple of small points: Greedy Score is referenced under informativeness measures included -> should this be changed to Mean Prediction Score? Also, under - examples showcasing different uncertainty estimator, `gpytorch_integration.py` is referenced. Should this be` model_gaussianprocesses.py`?

Thanks for catching this, adjusting now.

**Review by thomasgaudelet — APPROVED** [review]

**Review by alicerdv — APPROVED** [review]

**Pull request merged**